disk/datadog_checks/disk/disk.py

```diff
@@ -455,3 +466,26 @@ def _compile_tag_re(self):
             except TypeError:
                 self.log.warning('{} is not a valid regular expression and will be ignored'.format(regex_str))
         self._device_tag_re = device_tag_list
+
+    def _get_devices_label(self):
+        """
+        Get every label to create tags
+        """
+        devices_label = {}
+        try:
+            blkid_out, _, _ = get_subprocess_output(['blkid'], self.log)
+            all_devices = [l.strip().split() for l in blkid_out.splitlines()]
+
+            for d in all_devices:
+                # Line sample
+                # /dev/sda1: LABEL="MYLABEL" UUID="5eea373d-db36-4ce2-8c71-12ce544e8559" TYPE="ext4"
+                for att in d:
+                    if att.startswith('LABEL='):
+                        # Removing ':' from the name
+                        device_name = d[0][:-1]
+                        devices_label[device_name] = "label:{}".format(att[7:-1])
```

Comment:
```suggestion
                        devices_label[device_name] = "label:{}".format(att.split("=")[1])
```
Seems more readable that way. No big deal, feel free to disagree.